change to current api endpoint (/api/v1/states/daily.json); remove deprecated fields & alphabetize

liamcryan opened the pull request:
It looks like the updated API is:

 /api/v1/states/daily.json 

See https://covidtracking.com/data/api -> Historic values for all states.

I updated the fields: alphabetize to find a little easier, remove deprecated fields and included any newer ones.

Commit:
change to current api endpoint (/api/v1/states/daily.json); remove deprecated fields & alphabetize

stefanba3 reviewed (approved):
Nice! Thanks for doing this!

liamcryan committed:
update data source

COVID tracking API site will no longer provide data next month.  Change url to fixed .json file.

liamcryan committed:
change WDC connector location
